android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -15,6 +15,29 @@
 import android.os.Handler;
 import android.widget.Toast;
 
+/**
+ * TagDispatcher provides a unified, simple and exclusive interface for NFC.
```

Comment:
This is only the short explanation. The longer version is the lines after is: "While TagDispatcher is active ALL NFC activity will go directly to your implementation."